test/server/index.js

```diff
@@ -85,4 +85,4 @@ describe('applyContainerQuery', function () {
       .toBe('<div>{&quot;mobile&quot;:true,&quot;desktop&quot;:false}</div>');
   });
 
-});
+});
```

Comment:
I ran the code locally, and added a couple of tests. Good news: the code passed without modification.

Would you add the server rendering tests to your PR branch?

Here's what I did, pretty minimal, but it works if you want to copy-paste it.

```js
describe('useContainerQuery', function () {

  const query = {
    mobile: { maxWidth: 399 },
    desktop: { minWidth: 400 }
  };

  function renderContainerToStaticMarkup({ initialSize }) {
    const Container = () => {
      const [params, containerRef] = useContainerQuery(query, initialSize);

      return React.createElement(
        'div',
        {
          ref: containerRef
        },
        JSON.stringify(params)
      );
    };

    return renderToStaticMarkup(React.createElement(Container));
  }

  it('no matches with no initial widths', function () {
    const output = renderContainerToStaticMarkup({});
    expect(output).toBe('<div>{}</div>');
  });

  it('sets "mobile" to true at small sizes', function () {
    const zeroWidthOutput = renderContainerToStaticMarkup({ initialSize: { width: 399 } });
    const maximumMobileWidthOutput = renderContainerToStaticMarkup({ initialSize: { width: 0 } });

    expect(zeroWidthOutput).toBe('<div>{&quot;mobile&quot;:true,&quot;desktop&quot;:false}</div>');
    expect(maximumMobileWidthOutput).toBe('<div>{&quot;mobile&quot;:true,&quot;desktop&quot;:false}</div>');
  });

  it('sets "desktop" to true at large sizes', function () {
    const output = renderContainerToStaticMarkup({ initialSize: { width: 400 } });
    expect(output).toBe('<div>{&quot;mobile&quot;:false,&quot;desktop&quot;:true}</div>');
  });
});
```